ui/ducks/send/send.js

```diff
@@ -186,8 +183,16 @@ async function estimateGasLimitForSend({
   let isSimpleSendOnNonStandardNetwork = false;
 
   // blockGasLimit may be a falsy, but defined, value when we receive it from
-  // state, so we use logical or to fall back to MIN_GAS_LIMIT_HEX.
-  const blockGasLimit = options.blockGasLimit || MIN_GAS_LIMIT_HEX;
+  // state, so we use logical or to fall back to MIN_GAS_LIMIT_HEX. Some
+  // network implementations check the gas parameter supplied to
+  // eth_estimateGas for validity. For this reason, we set token sends
+  // blockGasLimit default to a higher number. Note that the current gasLimit
+  // on a BLOCK is 15,000,000 and will be 30,000,000 on mainnet after London.
+  // Meanwhile, MIN_GAS_LIMIT_HEX is 0x5208.
+  const blockGasLimit =
```

Comment:
Before this would be `options.blockGasLimit` if it was defined. Now it will be `GAS_LIMITS.BASE_TOKEN_ESTIMATE` if `options.blockGasLimit` is defined. To ensure backwards compatibility while addressing the issue this is meant to address, I might suggest:
```
let blockGasLimit = MIN_GAS_LIMIT_HEX;
if (options.blockGasLimit) {
blockGasLimit = options.blockGasLimit;
} else if (sendToken) {
blockGasLimit = GAS_LIMITS.BASE_TOKEN_ESTIMATE;
}
```

Reply:
if options.blockGasLimit is defined it'll be set to that due to the logical `||` operator. the asset type is only checking if the blockGasLimit is not provided

Reply:
I can make this change for legibility though.

Reply:
Done in latest